amm/src/main/scala/ammonite/Main.scala

```diff
@@ -66,7 +66,7 @@ case class Main(predef: String = "",
     )
   }
 
-  def instantiateInterpreter(replApi: Boolean) = {
+  def instantiateInterpreter(replApi: Boolean, verboseIvy: Boolean = true) = {
```

Comment:
This should be an argument to the `Main` object, rather than to each individual method